changelog.txt

```diff
@@ -7,22 +7,21 @@ The **signac** package follows `semantic versioning <https://semver.org/>`_.
 Version 2
 =========
 
-[2.2.0] -- 2024-xx-xx
+[2.2.0] -- 2024-02-13
 ---------------------
 
 Added
 +++++
 
  - Official support for Python 3.12 (#957).
- - ``Job.cached_statepoint`` - cached and read only access to job state points. Faster than
-   ``Job.statepoint`` (#975).
+ - ``Job.cached_statepoint`` - cached and read only access to job state points. Faster than ``Job.statepoint`` (#975).
```

Comment:
Sphinx appears to render your modification correctly: https://signac--984.org.readthedocs.build/projects/core/en/984/changes.html#id1

However, it is technically not valid rst syntax: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#bullet-lists .

Reply:
We can fix this in a new PR

Reply:
If it works, it may not be worth the change.

I quite often find lists like this:
```
- Some entry.
    Continued line.
```

Or:
```
1. List item.
    Continue line.
```
Which rst interprets as definition lists. This does not render as intended in HTML. Apparently, removing all the spaces does. Maybe there is something I missed in the spec about ignoring newlines if the next line does not start with white space?